### PR TITLE
Remove all SIL-specific stuff

### PR DESCRIPTION
**Pull request by kinke**
Moving it to SIL instead.

### CODE REVIEW

**Comment by kinke** [comment]
Still need to confirm `@SILdoc`.

**Comment by kinke** [comment]
No idea how it's working in detail, but `@SILdoc` is working as-is without depending on `sil-lang`. I've tested a full REPL:
```
> :? imap.Capability
imap.CapabilityEnumMembers
Capabilities of mail server
```

File: source/symmetry/imap/grammar.d [hunk 1]
```diff
@@ -6,8 +6,9 @@ module symmetry.imap.grammar;
     a PEG grammer.
 +/
 
-version (SIL) :
-    import pegged.grammar;
+version (Have_pegged):
```
Comment (kinke) [inline source/symmetry/imap/grammar.d:9]: This unused module was previously enabled for the SIL plugin and `sil-library` config. Not anymore - there's the `with-grammar` config now instead (build tested by CI).

**Comment by skoppe** [comment]
> No idea how it's working in detail, but `@SILdoc` is working as-is without depending on `sil-lang`.

Its an alias to some symbol in mir IIRC.

**Pull request merged**